### PR TITLE
Adding Powershell commands to Windows instructions

### PR DESCRIPTION
**Pull request by linsomniac**
I've added some Powershell commands to edit the registry as an option beyond using regedit.

- [X] read the [CONTRIBUTING guidelines](README.md#contributing)
- [ ] raised a GitHub issue or discussed it on the projects chat beforehand
- [ ] added unit tests
- [ ] added integration tests
- [X] updated documentation if needed
- [ ] updated CHANGELOG.md

### CODE REVIEW

File: docs/windows-client.md [hunk 1]
```diff
@@ -11,8 +11,17 @@ To make the Windows client behave as expected and to run well with `headscale`,
 - `HKLM:\SOFTWARE\Tailscale IPN\UnattendedMode` must be set to `always` as a `string` type, to allow Tailscale to run properly in the background
 - `HKLM:\SOFTWARE\Tailscale IPN\LoginURL` must be set to `<YOUR HEADSCALE URL>` as a `string` type, to ensure Tailscale contacts the correct control server.
 
+You can set these using the Windows Registry Editor:
+
 ![windows-registry](./images/windows-registry.png)
 
+Or via the following Powershell commands (right click Powershell icon and select "Run as administrator":
```
Comment (kradalby) [inline docs/windows-client.md:18]:
```suggestion
Or via the following Powershell commands (right click Powershell icon and select "Run as administrator"):
```

**Review by kradalby — COMMENTED** [review]
Looks good, would you be able to replicate this into https://github.com/juanfont/headscale/blob/main/templates/windows.html as well?

**Comment by linsomniac** [comment]
Thanks for pointing out that location, added in there as well.

**Comment by kradalby** [comment]
@linsomniac looks good, could you run the Prettier linter on the docs?

**Comment by linsomniac** [comment]
> @linsomniac looks good, could you run the Prettier linter on the docs?

Done.

**Review by kradalby — APPROVED** [review]

**Pull request merged**